resources/agent.rb

```diff
@@ -22,13 +20,6 @@
 property :elastic_agent_plugin_id, :kind_of => [ String, nil ], :required => false, :default => nil
 
 action :create do
-  if node['gocd']['agent']['go_server_host'] != nil || node['gocd']['agent']['go_server_port'] != nil
```

Comment:
Can we keep this warning or make it an error? It could say:
> Please not that node['gocd']['agent']['go_server_host'] and node['gocd']['agent']['go_server_port'] have no effect and have been replaced by node['gocd']['agent']['go_server_url']